### PR TITLE
Fix nav on Messages screen on mobile

### LINKED ISSUE #3068: Fixed Mobile Nav
On the conversations screen (`/messages`), the nav (hamburger menu + heading + avatar) is fixed to the top of the screen while everything else scrolls under it. I've not seen this behavior on any other of our DApp screens (including an individual conversation), and I think we prefer it (at least in our own app where there is no other top nav element). We may want to consider adding a horizontal rule so that the contents don't scroll into a white line of nothing. Regardless, let's apply this UX everywhere on small screens. 🎩

-----

At Origin, we use [ZenHub](https://www.zenhub.com/) to manage our engineering tasks and product development. [Download their browser extension](https://www.zenhub.com/extension) and check out our open workspace at [github.com/originprotocol/origin#zenhub](https://github.com/originprotocol/origin#zenhub).

### PR DESCRIPTION
**Pull request by norbertdragan**
### Description:

Fixes #3068 

The navbar at the Messages screen on Mobile is no longer fixed. I also noticed that the title for messages was not center aligned vertically (see [this](http://i.norbert.dev/868bf3)), yet it is on every other page, so I fixed that, too.

### Checklist:

- [X] Test your work and double-check to confirm that you didn't break anything
~- [ ] [Wrap any new text/strings with `fbt`](https://github.com/OriginProtocol/origin/tree/master/dapps/marketplace/translation#wrapping-text) for translation~
~- [ ] If there are any new/changed strings to translate, [`npm run translate`](https://github.com/OriginProtocol/origin/tree/master/dapps/marketplace/translation#integrating-translations)~
~- [ ] Update any relevant READMEs and [docs](https://github.com/OriginProtocol/origin/tree/master/docs)~

### CODE REVIEW

**Comment by micahalcorn** [comment]
Thanks @norbertdragan 😬 I was actually calling for the opposite: for the navbar to be fixed on all screens. That said, I suppose it still might be an improvement to have consistent behavior here even if it's not the ideal behavior.

**Comment by shahthepro** [comment]
@micahalcorn Didn't we end up removing fixed navbar on modals (mobile) since you(?) felt it looked weird with no border or shadow that separated content and the navbar? I remember doing this for all modals while we changed them from dark to light themed.

**Comment by micahalcorn** [comment]
@shahthepro I don't know about modals, specifically, but I do agree that we will need a line at the bottom of the nav for that reason. I mentioned that in the issue and realize that we will need to figure out how this will work on the home screen with the search bar.

File: dapps/marketplace/src/pages/messaging/Messages.js [hunk 1]
```diff
@@ -223,7 +223,6 @@ require('react-styl')(`
     .conversations-wrapper
       display: flex
       flex-direction: row
-      height: calc(100vh - 6rem)
```
[the remark this answers is not in the document]
Comment (norbertdragan) [reply dapps/marketplace/src/pages/messaging/Messages.js]: Right, it does effect the desktop behaviour. I should achieve this effect via a media query instead.

**Comment by nick** [comment]
btw, I have just merged a fix in the master branch that should enable tests to run on PRs from non-OriginProtocol organization members. Please merge in the latest master to see if the tests pass 😊

**Comment by norbertdragan** [comment]
@micahalcorn My bad, I understood the desired behavior the other way around. 🤦‍♂ I will work on that, then.

**Comment by norbertdragan** [comment]
Btw I noticed that the bottom controls for the chat window behave differently with the current build. In portrait mode on mobile, the controls are not sticking to the bottom of the screen - while in landscape mode and desktop, they do. I think it should be sticky in every scenario.

**Portrait mobile:**
The user needs to scroll down to unveil the controls



**Landscape mobile/desktop:** 
The user can see the controls at all times, instead, the messages need scrolling.

File: dapps/marketplace/src/pages/messaging/Messages.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (norbertdragan) [reply dapps/marketplace/src/pages/messaging/Messages.js]: 0467f67 resolves this issue.

**Pull request closed without merging**